src/test/java/ru/mipt/data/service/WorkoutServiceTest.java

```diff
@@ -260,18 +260,17 @@ void count() {
     @Test
```

Comment:
Tests run: 10, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.079 s <<< FAILURE! - in ru.mipt.data.service.WorkoutServiceTest
updateRating  Time elapsed: 0.038 s  <<< FAILURE!
org.opentest4j.AssertionFailedError: expected: <Optional[Workout{id=0, title='test', description='test', rating=5.0, workoutTags=[]}]> but was: <4.0>
	at ru.mipt.data.service.WorkoutServiceTest.updateRating(WorkoutServiceTest.java:273)